contracts/USM.sol

```diff
@@ -131,4 +148,75 @@ contract USM is BufferedToken {
         latestFumPrice = fumPrice();
         emit LatestFumPriceChanged(previous, latestFumPrice);
     }
+
+    /** INTERNAL FUNCTIONS */
+
+    /**
+     * @notice Funds the pool with ETH, minting FUM at its current price
+     */
+    function _fund(address to, uint ethIn) internal {
+        uint _fumPrice = fumPrice();
+        uint fumOut = ethIn.wadDiv(_fumPrice);
+        ethPool = ethPool.add(ethIn);
+        fum.mint(to, fumOut);
+        _setLatestFumPrice();
+    }
+
+    /**
+     * @notice Mint USM from Eth. Inheriting contract must handle the Ether transfers.
+     */
+    function _mint(uint ethAmount) internal returns (uint) {
+        uint usmAmount = _ethToUsm(ethAmount);
+        ethPool = ethPool.add(ethAmount);
+        super._mint(msg.sender, usmAmount);
+        return usmAmount;
+    }
+
+    /**
+     * @notice Burn USM for Eth. Inheriting contract must handle the Ether transfers.
+     */
+    function _burn(uint usmAmount) internal returns (uint) {
+        uint ethAmount = _usmToEth(usmAmount);
+        ethPool = ethPool.sub(ethAmount);
+        super._burn(msg.sender, usmAmount);
+        return ethAmount;
+    }
+
+    /**
+     * @notice Convert ETH amount to USM using the latest price of USM
+     * in ETH.
+     *
+     * @param _ethAmount The amount of ETH to convert.
+     * @return The amount of USM.
+     */
+    function _ethToUsm(uint _ethAmount) internal view returns (uint) {
+        if (_ethAmount == 0) {
+            return 0;
+        }
+        return _oraclePrice().wadMul(_ethAmount);
+    }
+
+    /**
+     * @notice Convert USM amount to ETH using the latest price of USM
+     * in ETH.
+     *
+     * @param _usmAmount The amount of USM to convert.
+     * @return The amount of ETH.
+     */
+    function _usmToEth(uint _usmAmount) internal view returns (uint) {
+        if (_usmAmount == 0) {
+            return 0;
+        }
```

Comment:
This check seems superfluous to me.  If they're silly enough to pass in 0 they can probably live with the same gas cost as a non-0 input.  Wdyt?

Reply:
Agreed, not needed